packages/tags/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@zendeskgarden/react-tags",
+  "description": "Components relating to tags in the Garden Design System",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "license": "UNLICENSED",
+  "repository": "https://github.com/zendeskgarden/react-components",
+  "bugs": {
+    "url": "https://github.com/zendeskgarden/react-components/issues"
+  },
+  "version": "0.0.0",
+  "main": "./index.js",
+  "scripts": {
+    "build": "../../utils/scripts/build.sh",
+    "build:styleguide": "../../utils/scripts/build-styleguide.sh",
+    "start": "../../utils/scripts/start.sh"
+  },
+  "dependencies": {
+    "@zendeskgarden/react-theming": "^1.0.0",
+    "@zendeskgarden/css-tags": "^4.0.0",
+    "classnames": "^2.2.5",
+    "prop-types": "^15.6.0",
+    "styled-components": "^3.2.5"
+  },
+  "devDependencies": {
+    "@zendeskgarden/react-selection": "^3.0.0",
+    "@zendeskgarden/react-textfields": "^2.0.0"
+  },
+  "peerDependencies": {
+    "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
+    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0"
+  },
+  "keywords": [
+    "components",
+    "garden",
+    "react",
+    "zendesk"
+  ]
+}
```

Comment:
Missing `publishConfig` here. Compare to https://github.com/zendeskgarden/react-components/blob/master/packages/.template/package.json and make sure this package.json is analogous.

Reply:
Updated all new fields in the template.